### ISSUE TITLE
Material design for form-controls

### ISSUE THREAD

**Issue by thehitechpanky** [twbs/bootstrap#30619]
Is it possible to include material design in form-control elements like textbox, textarea, etc. The design depicted on https://fezvrasta.github.io/bootstrap-material-design/docs/4.0/bootstrap-components/forms/#form-controls is cleaner for mobile layouts.

Rather than full rectangle, having just an underline for textbox is much more lighter for user experience in some cases.

**Comment by mdo** [comment]
If you want that, you can use it as `bootstrap-material-design` is another project separate from us. No plans to make Material any kind of a default here.

**Comment by thehitechpanky** [comment]
I was not commenting on whether or not to make it default. I'm simply requesting for an additional class like "material-design" which can change the css style of boxes to the one without borders and with underline.

**Comment by ffoodd** [comment]
Just to mention that Google finally user-tested their Material fields and they learned a few things: https://medium.com/google-design/the-evolution-of-material-designs-text-fields-603688b3fe03

TL;DR: they now use a light gray `background`, which kinda defeat the whole point. And is doable in Bootstrap pretty easily :)
